images/135_OpenShift_Distributed_Unit.svg

```diff
@@ -0,0 +1 @@
+<svg id="a9beebe0-72e4-44c9-ad10-dece272880e4" data-name="artwork" xmlns="http://www.w3.org/2000/svg" width="760" height="217.136"><defs><style>.bd106038-b07d-487e-80ba-7cf5527b8155,.ed46db36-8d71-4ee3-bb02-8a24d75bb940{fill:#151515}.a61cc133-0d96-4306-a078-c193f507e799{fill:#fff}.ed46db36-8d71-4ee3-bb02-8a24d75bb940{font-size:11px;font-family:RedHatText,&quot;Red Hat Text&quot;,Overpass,&quot;Helvetica Neue&quot;,Arial,sans-serif;font-weight:500}.b563919e-f7f3-40fc-8840-09c733f4cb30{fill:none;stroke:#151515;stroke-linecap:round;stroke-linejoin:round}</style></defs><path fill="#e8e8e8" d="M0 20h620v180H0z"/><text transform="translate(25 56.788)" fill="#151515" font-size="14" font-family="RedHatText,&quot;Red Hat Text&quot;,Overpass,&quot;Helvetica Neue&quot;,Arial,sans-serif" font-weight="700">RAN boundary</text><path class="a61cc133-0d96-4306-a078-c193f507e799" transform="rotate(-180 495 75)" d="M395 45h200v60H395z"/><text class="ed46db36-8d71-4ee3-bb02-8a24d75bb940" transform="translate(416.742 77.547)">Centralized Unit, control plane</text><path class="a61cc133-0d96-4306-a078-c193f507e799" transform="rotate(-180 495 145)" d="M395 115h200v60H395z"/><text class="ed46db36-8d71-4ee3-bb02-8a24d75bb940" transform="translate(422.836 147.547)">Centralized Unit, user plane</text><text transform="translate(671.531 200.153)" font-size="10" fill="#f3f3f3" font-family="RedHatText,&quot;Red Hat Text&quot;,Overpass,&quot;Helvetica Neue&quot;,Arial,sans-serif">135_OpenShift_0121</text><path fill="none" d="M0 177.136h760v40H0z"/><path class="b563919e-f7f3-40fc-8840-09c733f4cb30" d="M385.324 75H370v27.5h-17.824"/><path class="bd106038-b07d-487e-80ba-7cf5527b8155" d="M383.865 70.014L392.5 75l-8.635 4.986v-9.972zM353.635 97.514L345 102.5l8.635 4.986v-9.972z"/><path class="b563919e-f7f3-40fc-8840-09c733f4cb30" d="M385.324 145H370v-27.5h-17.824"/><path class="bd106038-b07d-487e-80ba-7cf5527b8155" d="M383.865 140.014L392.5 145l-8.635 4.986v-9.972zM353.635 112.514L345 117.5l8.635 4.986v-9.972z"/><g><path class="b563919e-f7f3-40fc-8840-09c733f4cb30" d="M192.837 110h-18.148"/><path class="bd106038-b07d-487e-80ba-7cf5527b8155" d="M191.378 105.014l8.635 4.986-8.635 4.986v-9.972zM176.148 105.014L167.513 110l8.635 4.986v-9.972z"/></g><path class="a61cc133-0d96-4306-a078-c193f507e799" transform="rotate(-180 272.5 110)" d="M202.5 80h140v60h-140z"/><text class="ed46db36-8d71-4ee3-bb02-8a24d75bb940" transform="translate(231.118 112.547)">Distributed Unit</text><path class="a61cc133-0d96-4306-a078-c193f507e799" transform="rotate(-180 95 110)" d="M25 80h140v60H25z"/><text class="ed46db36-8d71-4ee3-bb02-8a24d75bb940" transform="translate(68.139 112.547)">Radio Unit</text></svg>
```

Comment:
I see two image files, but only one embedded. Is that intended? If one of these files is outdated, we should remove it.

Reply:
I removed the outdated profile on my local repo,